test-harness/src/test/groovy/com/netflix/conductor/test/base/AbstractSpecification.groovy

```diff
@@ -61,4 +61,8 @@ abstract class AbstractSpecification extends Specification {
 
         workflowExecutor.startWorkflow(input)
     }
+
+    protected String startWorkflow(StartWorkflowInput startWorkflowInput) {
```

Comment:
nit: instead of this method in the base class, the sub-classes can directly call `workflowExecutor.startWorkflow()`. It adds a layer of indirection, which is not necessary.